.github/workflows/release.yaml

```diff
@@ -28,19 +20,12 @@ jobs:
 
       - run: yarn install --frozen-lockfile
 
-      - name: Bump version
-        run: |
-          git config --global user.email "j@arcx.money"
-          git config --global user.name "ARCx"
-          npm version ${{ github.event.inputs.version }}
-
       - name: Push latest vestion
         run: git push origin main --follow-tags
 
-      - name: Release package
+      - name: Semantic Release
         run: |
           yarn build
-          npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
-          npm publish
+          yarn semantic-release
```

Comment:
Have you had chance to test it out?

Reply:
Good call, I tested it out and it worked. It turns out if you run it via the CLI it runs in _dry run_ mode which doesn't actually publish the package, but you get to know what version it would publish at and what the changelog looks like. For example:

![image](https://user-images.githubusercontent.com/5834876/203385597-23f993e2-88b8-46a5-9c6c-ad3108005a64.png)

Reply:
That means that when we merge PRs we don't have to actively remove commit messages that are not useful to go in the changelog, as the commits other than `BREAKING CHANGE:, feat:, fix:` will be discarded.